src/ui/stats.js

```diff
@@ -34,55 +38,53 @@ export class StatsScene extends Phaser.Scene {
     create() {
         // Gold
         this.goldText = this.add
-            .text(20, 20, "Gold: " + this.characterData.gold, {
-                fill: "gold",
-                fontSize: "30px",
-            })
+            .text(20, 20, "Gold: " + this.characterData.gold, FONTS.GOLD)
             .setDepth(3);
 
         // Show stats
-        let statColor = "white";
         this.enemiesKilledText = this.add
-            .text(20, 60, "Enemies killed: " + this.characterData.totalEnemiesKilled, {
-                fill: statColor,
-            })
+            .text(20, 60, "Enemies killed: " + this.characterData.totalEnemiesKilled, FONTS.STATS)
             .setDepth(3);
         this.timesClickedText = this.add
-            .text(20, 75, "Times clicked: " + this.characterData.timesClicked, {
-                fill: statColor,
-            })
+            .text(20, 75, "Times clicked: " + this.characterData.timesClicked, FONTS.STATS)
             .setDepth(3);
         this.damageByClickingText = this.add
             .text(
                 20,
                 90,
                 "Damage done by clicking: " + this.characterData.damageByClicking,
-                {
-                    fill: statColor,
-                }
+                FONTS.STATS
             )
             .setDepth(3);
+        this.clickDpsText = this.add
+            .text(20, 75, "Click DPS: 0", FONTS.STATS)
+            .setDepth(3);
         this.damageByAutoClickText = this.add
             .text(
                 20,
                 105,
                 "Damage done by autoclickers: " + this.characterData.damageByAutoClick,
-                {
-                    fill: statColor,
-                }
+                FONTS.STATS
             )
             .setDepth(3);
         this.autoClickDpsText = this.add
-            .text(20, 120, "AutoClicker DPS: " + this.autoClickDps, {
-                fill: statColor,
-            })
+            .text(20, 120, "AutoClicker DPS: " + this.autoClickDps, FONTS.STATS)
```

Comment:
idk what is causing it but there is overlap in the text on the Lumbridge woodcutting level

Reply:
Forgot to add the visibility setter in my show/hide function for the new DPS tracker. Fixed